MainRepository/Program.cs

```diff
@@ -1,7 +1,35 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
-}
\ No newline at end of file
+﻿// Program to convert the decimal number to binary and hexadecimal number.
+Console.Write ("Enter the number: ");
+string input = Console.ReadLine ();
+List<int> binary = new ();
+List<char> hexadecimal = new ();
+if (int.TryParse (input, out int number) && number > 0) {
+   while (number > 0) {
+      binary.Add (number % 2);
+      number /= 2;
+   }
+   number = int.Parse (input);
+   while (number > 0) {
+      int temp = number % 16;
+      if (temp < 10) {
+         // if temp value range from 0 to 9, then we have to add 48 to convert integer value to ascii character.  

```

Comment:
Mention why you used 48. Why this magic number? Similarly add comments for 55. From the code it is clear you are switching based on temp. What is unclear is why 48 and 55?